compose bashrc customizations

craigsteffen opened the pull request:
Starting new branch for the file customize bashrc.  Making a branch and PR so that I can work on this formatting and customizations until they're ready for proposed_changes.

craigsteffen commented:
@rmokos @e-eight @greghbauer @wjackson-gh 
This don't strictly need a review, but this bashrc customization page is a bunch of new-ish stuff, so I'd prefer to have someone look at it and confirm that it's a reasonable start.  

I think if you click on the PR link, there's a green button in the upper right to submit your review (which is text, and also a check box for "approve" or not).

e-eight commented on docs/source/common/customize_bashrc.rst:
```diff
@@ -0,0 +1,70 @@
+------------------
+It's useful to add often-used aliases to your .bashrc file.  Common shell aliases include: 
+
+rm="rm -i"
```
Recommend putting these in inline code elements

pmenstrom commented on docs/source/common/customize_bashrc.rst:
Add "is" after PATH
PATH is a colon-separated list.

pmenstrom commented on docs/source/common/customize_bashrc.rst:
Missing a period after PATH
When the shell starts up it has a sensible default list in PATH. If you have a directory with scripts of your own, it’s handy to add that directory to your path.

pmenstrom commented on docs/source/common/customize_bashrc.rst:
equal should be singular
his sets the new PATH equal to the existing path ($PATH)

pmenstrom commented on docs/source/common/customize_bashrc.rst:
you're is misspelled
This is very handy so that you can review by eye the file that youre

craigsteffen commented on docs/source/common/customize_bashrc.rst:
```diff
@@ -0,0 +1,70 @@
+
+Modifying your path
+----------------------
+Shells have an internal variable called PATH that's a list of directories where bash will look for commands to run.  PATH a colon-separated list.  When the shell starts up it has a sensible default list in PATH  If you have a directory with scripts of your own, it's handy to add that directory to your path.  This is also true if you install software in your own space on a system; you'll usually w ...
```
Yes.  Creating a commit now to fix.

pmenstrom commented on docs/source/common/customize_bashrc.rst:
"she" should be "the"
 If you put this clause in your .bashrc, then if you start an interactive job, she new shell

pmenstrom reviewed (changes requested):
I left some comments. They are all small spelling or grammar changes. It looks good other than that.

craigsteffen commented:
@e-eight Thanks for all the notes. I've implemented all those except the -f -i example.  

I've also implemented all of @pmenstrom 's corrections and updates. 

I'll merge this into proposed_changes unless someone objects quickly.
